.changeset/small-bugs-prove.md

```diff
@@ -0,0 +1,34 @@
+---
+'astro': minor
+---
+
+Order Astro styles last, to override imported styles
+
+This fixes CSS ordering so that imported styles are placed *higher* than page/component level styles. This means that if you do:
+
+```astro
+---
+import '../styles/global.css';
+---
+<style>
+  body {
+    background: limegreen;
+  }
+</style>
+```
+
+The `<style>` defined in this component will be placed *below* the imported CSS. When compiled for production this will result in something like this:
```

Comment:
Just to double check here, these will both be added *after* any user provided `<link rel="stylesheet">` in the head, right?  Meaning any stylesheets pulled from `/public` or an external CDN would be first on page?

Reply:
Yes, all of the CSS and scripts we add are added to the bottom of the head, so after all user-defined head content. We don't parse the HTML so we don't even know about your link tags.